docs/recipes/php_composer.md

```diff
@@ -0,0 +1,73 @@
+# 🐘 Using semantic-release to release a PHP/Composer package
+
+You can use semantic-release to release a [Composer](https://getcomposer.org/) package to [packagist.org](https://packagist.org/).
+Composer works with GitHub repositories, so all that needs to happen is tag a new release on GitHub.
+The last release can be figured out by looking at the git tags of the repository.
+
+### Example annotated `package.json`
+
+```js
+{
+  "scripts": {
+    "semantic-release": "semantic-release"
+  },
+  "release": {
+    // Configure verifyConditions to not include condition-travis (we don't need it because we use Travis build stages)
```

Comment:
In order to make this recipe simpler and more focused on `Composer` maybe it be better to not mix Travis Build Stage and keep the `conditions-travis` plugin.
Not sure if it's possible though. I think node and npm are installed by default on each travis images, so you might be able to do something like that:
```yaml
php:
  - '7.0'
  - '7.2'

install:
  - composer install --prefer-dist
script:
  - vendor/bin/phpunit

after_success:
  - npx semantic-release
```

Reply:
Not Node 8 though, you'd have to run `nvm install 8; nvm use 8` first, and then `npm install`. I think using build stages lowers the barrier especially for any non-JS project because you can just "drop in" that totally isolated release stage that is completely independent of the rest of your build config.

Reply:
ok, works for me